conf/cloud-region/eu_west_2.config

```
@@ -0,0 +1,3 @@
+params {
+    igenomes_base     = 's3://reference-data-cyn-pro/igenomes'
```

```suggestion
    reference_data_bucket    = 's3://reference-data-cyn-pro/igenomes'
```
if we need to keep this for cynapxe, then maybe it would be best to have it as reference_data_bucket